sfxclient/httpsink.go

```diff
@@ -450,6 +490,25 @@ func sapmMarshal(v []*trace.Span) ([]byte, error) {
 	return proto.Marshal(msg)
 }
 
+func parseRetryAfterHeader(v string) (time.Duration, error) {
+	if v == "" {
+		return 0, errors.New("empty header value")
+	}
+
+	// Retry-After: <http-date>
+	// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date
+	if t, err := http.ParseTime(v); err == nil {
+		return t.Sub(time.Now()), nil
+	}
+
+	// Retry-After: <delay-seconds>
```

Comment:
It looks like this form is the only one we emit, not the (above, and likely correct) HTTP 1.1 date.

Reply:
(just a note, that you included the date parse above is nice of you!)